x-pack/packages/kbn-elastic-assistant-common/impl/schemas/knowledge_base/entries/bulk_crud_knowledge_base_entries_route.http

```diff
@@ -0,0 +1,13 @@
+### Empty Bulk Action
+POST http://{{host}}:{{port}}{{basePath}}/internal/elastic_assistant/knowledge_base/entries/_bulk_action
+kbn-xsrf
+Elastic-Api-Version: 1
+Content-Type: application/json
+Authorization: Basic elastic changeme
```

Comment:
Should this also be a variable?

Reply:
So for having the headers as variables, I had looked into it as for most requests we don't really care about the headers (just the URL/payload and then response), so it would be nice if we could collapse them to keep the file cleaner, but seems it's only possible currently to abstract the header values as a [variable](https://www.jetbrains.com/help/idea/exploring-http-syntax.html#using_request_vars), not all the headers themselves (which maybe we don't want anyway as that then ties auth to the rest of the headers to a specific variable?). 

This is all a trial before going down any `.http` file generation route from OAS, so we can see what's more ergonomic before building this out further. I would also like to import payloads as the same mocks we use in our API integration tests, then maybe we can generate mocks (via LLM) as part of the OAS generation workflow. Then at that point from a spec you can get all the types, mocks, and http clients, which would be really nice for streamlining API development. cc @maximpn 

In the meantime, I did at least pull the header values out into variables in https://github.com/elastic/kibana/pull/186566/commits/a0de4d17e61a99a484e5f34650309057f3be5586. Thanks for the recommendation @jbudz!